core/src/main/java/hudson/os/PosixException.java

```diff
@@ -2,7 +2,6 @@
 
 /**
  * Indicates an error during POSIX API call.
- * @see PosixAPI
  * @author Kohsuke Kawaguchi
  */
 public class PosixException extends RuntimeException {
```

Comment:
Hmm, still used by `IOUtils.mode`. Wonder if that can be converted to use `IOException`.

Reply:
Yeah probably.